resources/rules/post_identifier_merge/temp/drugbank-skos/drugbank_skos_exact_match_mapping.clj

```diff
@@ -0,0 +1,23 @@
+;; -----------------------------------------------------
+;; --------- DrugBank SKOS Exact Match Mapping ---------
+;; -----------------------------------------------------
+`{:name "drugbank-drug-exact-mapping-assertion"
+  :description "This rule identifies drug concept exact matches for every drug record and types them as (IAO_EXT_0000404)"
+  :head (
+          ;; create an exact match to drugbank identifier
+          (?/drug_ice_id skos/exactMatch ?/other_drug_ice_id))
+
+  :body ((?/record rdf/type ccp/IAO_EXT_0000426) ; ccp:DrugBank_drug_record
+         (?/record obo/BFO_0000051 ?/drug_field_value) ; BFO:has_part
+         (?/drug_field_value rdf/type ccp/IAO_EXT_0000360) ; ccp:Drugbank_drug_record__drugbank_identifier_field_value 
+         (?/drug_field_value rdf/type ?/drug_ice_id)
+
+         ;; retrieve the external exact match identifier
+         (?/record obo/BFO_0000051 ?/other_drug_field_value) ; BFO:has_part
+         (?/other_drug_field_value rdf/type ccp/IAO_EXT_0000404) ; ccp:Drugbank_drug_record__data_source_identifier_field_value
+         (?/other_drug_field_value obo/IAO_0000219 ?/other_drug_ice_id)) ; IAO:denotes
+
```

Comment:
This comment is a placeholder for future work. Ideally here we would use the identifier hierarchy to check to make sure the two identifiers can be exact matches of each other. For this rule, we might want to check that the other_drug_ice_id is also rdf:type 'drug identifier', or maybe rdf:type 'chemical identifier'.  That hierarchy is not present in the ccp-extension ontology currently, which is why this is a placeholder for work in the future.